OpenRA.Mods.Common/Traits/SupportPowers/NukePower.cs

```diff
@@ -94,7 +94,11 @@ public override void Activate(Actor self, Order order, SupportPowerManager manag
 			base.Activate(self, order, manager);
 
 			if (self.Owner.IsAlliedWith(self.World.RenderPlayer))
+			{
 				Game.Sound.Play(Info.LaunchSound);
+				Game.Sound.PlayNotification(self.World.Map.Rules, self.Owner, "Speech",
```

Comment:
We have that at https://github.com/OpenRA/OpenRA/pull/11560/files#diff-3c45b822e980db7199501bd00ec7beb6R104.